File: ojuz/ioi/2020/Day1/tickets_sol/public/cpp/tickets.cpp
#include "tickets.h"
#include <vector>

long long find_maximum(int k, std::vector<std::vector<int>> x) {
	int n = x.size();
	int m = x[0].size();
	std::vector<std::vector<int>> answer;
	for (int i = 0; i < n; i++) {
		std::vector<int> row(m);
		for (int j = 0; j < m; j++) {
			if (j < k) {
				row[j] = j;
			} else {
				row[j] = -1;
			}
		}
		answer.push_back(row);
	}
	allocate_tickets(answer);
	return 1;
}

File: ojuz/ioi/2020/Day1/tickets_sol/public/java/tickets.java
public class tickets {
	long find_maximum(int k, int[][] x) {
		int n = x.length;
		int m = x[0].length;
		int[][] answer = new int[n][m];
		for (int i = 0; i < n; i++) {
			for (int j = 0; j < m; j++) {
				if (j < k) {
					answer[i][j] = j;
				} else {
					answer[i][j] = -1;
				}
			}
		}
		grader.allocate_tickets(answer);
		return 1;
	}

}

